### DISCUSSION TITLE
Can you add a gas water heater?

### DISCUSSION THREAD

**Discussion by zgp0126** [ofalvai/home-assistant-candy#43] [Device support improvements]
Model:JSQ31-16M7X（12T）

**Comment by ofalvai** [comment]
@zgp0126 I can add support, but I need some help with how this device works.

Please check its API response with [this tool](https://github.com/MelvinGr/CandySimplyFi-tool) and paste some API responses in different device states. Eventually, we'll need to decode the meaning of each field in the API response. You can see how this was done with other appliances here: https://github.com/ofalvai/home-assistant-candy/blob/main/custom_components/candy/client/model.py
